CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # see https://help.github.com/en/articles/about-code-owners#example-of-a-codeowners-file
 
-* @pagopa-tech
+* @pagopa-tech @jacopocarlini @pasqualespica @aomegax
```

Comment:
Why this changes ?
To manage [pr routing-algorithms](https://docs.github.com/en/enterprise-server@3.2/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#routing-algorithms) ?